dev_guide/builds/jenkins_pipeline_builds.adoc

```diff
@@ -0,0 +1,169 @@
+[[dev-guide-jenkins-pipeline-builds]]
+= Jenkins Pipeline Builds
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[overview]]
+
+== Overview
+
+
+
+[[how-it-works]]
+
+== How It Works
+
+
+[ditaa, "jenkins-pipeline-lifecycle"]
+----
++----------------------------------+
+|                                  |
+|   A Pipeline build is triggered  |
+|   in OpenShift via command line, |--------------------------------+
+|   web console, or API            |                                |
+|                                  |                                |
++----------------------------------+                                |
+                                                                    v
+                                     +------------------------------------------------------------+
+                                     |                                                            |
+                                     |   The Jenkins Master for your project observes that a new  |
+                +--------------------|   Pipeline build object has been created. A new Pipeline   |
+                |                    |   job is then created and a new Jenkins Slave is requested |
```

Comment:
jenkins slave is optional, only requested if the pipeline wants to run on a slave...